packages/engine/src/output/utils.rs

```diff
@@ -2,14 +2,12 @@ use memory::shared_memory::MemoryId;
 
```

Comment:
The todo wanted to "move this to top-level", I'm not sure if it's referring to the crate root or the module level since this is kinda related to output.